package.json

```diff
@@ -12,8 +12,7 @@
     "@smartthings/smartapp": "^1.0.3",
     "axios": "^0.19.2",
     "body-parser": "*",
-    "chai": "^4.2.0",
-    "chai-http": "^4.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v1.0.0",
```

Comment:
This will need to be updating to the correct version after this PR is merged: https://github.com/bravetechnologycoop/brave-alert-lib/pull/3

Reply:
Could we perhaps specify a ^1.0.0, so that we don't need to update it? Is that the version you were referring to Theresa?

Reply:
Unfortunately not. When we use the git way of doing packages, we can't use any of the semantic versioning niceness. The `v1.0.0` in this case is just the string name of the tag, it doesn't carry with it any of the logic like the ones that grab from npm itself.

Reply:
Will do in https://github.com/bravetechnologycoop/ODetect-Backend-Local/pull/57